packages/devkit/src/utils/log-show-project-command.ts

```diff
@@ -0,0 +1,14 @@
+import { requireNx } from '../../nx';
+
+export function logShowProjectCommand(projectName: string): void {
+  if (process.env.NX_PCV3 !== 'true') {
+    return;
+  }
```

Comment:
This works w/o PCV3 though right? I don't know if its worth feature-flagging this

Reply:
Good point, yes it does.